exercises/concept/card-tricks/.docs/instructions.md

```diff
@@ -4,70 +4,89 @@ As a magician-to-be, Elyse needs to practice some basics. She has a stack of car
 
 To make things a bit easier she only uses the cards 1 to 10.
 
-## 1. Retrieve a card from a stack
+## 1. Create a slice with certain cards
```

Comment:
Headers in this file and in hints need to match, otherwise the website has issues to connect the "show hints" button to the correct content.

Reply:
Done.

Reply:
The header should be a sentence like "Create a slice with certain cards" as you had it before in instructions, now you changed it to be the function name (like it was in hints).

Reply:
Done.